frontend/packages/core/src/AppProvider/short-link.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import type { clutch as IClutch } from "@clutch-sh/api";
+
+import { useNavigate } from "../navigation";
+import { client } from "../Network";
+import type { ClutchError } from "../Network/errors";
+
+/**
+ * The base for a short link route
+ */
+export const ShortLinkBaseRoute = "sl";
```

Comment:
Yeah goto would be fine. We could build it off an environment variable if we wanted, default to `goto` but if something else is available it could use that

Reply:
Adjusted to use `goto`, we can expand this for an environment variable in the future though